src/class/hid/hid_device.c

```diff
@@ -418,7 +418,8 @@ bool hidd_control_request(uint8_t rhport, tusb_control_request_t const * p_reque
 
     if (p_request->bRequest == TUSB_REQ_GET_DESCRIPTOR && desc_type == HID_DESC_TYPE_REPORT)
     {
-      usbd_control_xfer(rhport, p_request, p_hid->desc_report, p_hid->desc_len);
+      // Cast away the const on p_hid->desc_report because we know it won't be modified.

```

Comment:
ah right, forgot about this, hmm, I need to raise my ses warning level to catch this.